_patternlab-files/index.twig

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html id="patternlab-html">
+<head id="patternlab-head">
+	<title id="title">Pattern Lab</title>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width" />
+	<link rel="stylesheet" href="styleguide/css/styleguide.css" media="all" />
+	<link rel="stylesheet" href="styleguide/css/styleguide-specific.css" media="all" />
+	<link rel="stylesheet" href="styleguide/css/vendor/typeahead.css" media="all" />
+	<link rel="stylesheet" href="styleguide/css/vendor/prism.css" media="all" />
+	<script src="http://ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
```

Comment:
PL Wrapper requires jquery, it takes some time to figure out how to make it working without it